docs/docs/components/reanimated_swipeable.md

```diff
@@ -80,6 +80,16 @@ Receives swipe direction as an argument.
 a function that is called when `swipeable` starts animating on close.
 Receives swipe direction as an argument.
 
+### `onSwipeableOpenStartDrag`
+
+a function that is called when `swipeable` starts being dragged open.
+Receives swipe direction as an argument.
+
+### `onSwipeableCloseStartDrag`
+
+a function that is called when `swipeable` starts being dragged close.
```

Comment:
```suggestion
a function that is called when a user starts to drag the `swipable` to close.
```

Reply:
fixed in [665da69](https://github.com/software-mansion/react-native-gesture-handler/pull/3150/commits/665da69375e5e4e3e7ce11262e07fc2d8cec92e9)